pkgs/top-level/python-packages.nix

```diff
@@ -8632,11 +8634,11 @@ in {
 
   ConfigArgParse = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix.`

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };.`